### PR TITLE
[12.0][FIX] contract: errors in navigation to customer invoices / vendor bills

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Forward port of https://github.com/OCA/contract/pull/273.

### CODE REVIEW

**Review by carlosdauden — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
Tested

**Comment by rafaelbn** [comment]
Waiting to see travis....

File: contract/models/account_analytic_account.py
Comment (nikul-serpentcs) [inline contract/models/account_analytic_account.py:358]: @mreficent  Improved method code, No need to Else part

```
action = self.env.ref(
            'contract.act_purchase_recurring_invoices')
        if self.contract_type == 'sale':
            action = self.env.ref(
                'contract.act_recurring_invoices')
```

**Review by nikul-serpentcs — APPROVED** [review]
Code review LGTM :+1: 

Minor Change as per MO

**Comment by MiquelRForgeFlow** [comment]
Ready to merge :green_apple:

**Comment by AaronHForgeFlow** [comment]
As the most important fix here is already included in #207, then perhaps it is better to wait until #207 is merged. Then we can rebase this PR.

**Comment by MiquelRForgeFlow** [comment]
Deleted manifest number version update to avoid conflicts with the other v12 PR that also will be merged soon.

**Pull request merged**
